src/lib/slack/commands/ThingsBotCommand.js

```diff
@@ -19,18 +19,18 @@ const ThingsBotCommand = {
             const deviceList = await getDeviceList(this.installedSmartAppId);
 
             if (!deviceList) {
-                console.error('ERROR ** Failed to pull valid device list in deviceCommandsBlock getter')
+                console.error('ERROR ** Failed to pull valid device list in deviceCommandsBlock getter');
                 return {
                     "response_type": "ephemeral",
                     "text": "Uh oh, there was a problem pulling your device list. Please try again."
                 };
-            } else if (deviceList.length == 0) {
+            } else if (deviceList.length === 0) {
```

Comment:
Any particular reason for switching to triple equals here? `.length` is always going to return a `Number`, so allowing type conversion isn't an issue. Also if for some reason it returned something that converted to `0` I think we'd still want to trip the no devices response rather than sending a broken normal response.

Reply:
WebStorm was throwing some yellow my way so I just made the recommended edit, I can remove tho. I was just listening to the tooling here